ts/hooks/use_stake.ts

```diff
@@ -47,16 +47,14 @@ export const useStake = (networkId: ChainId, providerState: ProviderState): UseS
     const [ownerAddress, setOwnerAddress] = useState<string | undefined>(undefined);
     const [stakingContract, setStakingContract] = useState<StakingContract>(undefined);
     const [stakingProxyContract, setStakingProxyContract] = useState<StakingProxyContract | undefined>(undefined);
-    const [contractAddresses, setContractAddresses] = useState<ContractAddresses | undefined>(undefined);
 
     useEffect(() => {
         const _ownerAddress = (providerState.account as AccountReady).address;
         const _contractAddresses = getContractAddressesForChainOrThrow(networkId);
 
-        setContractAddresses(_contractAddresses);
         setOwnerAddress(_ownerAddress);
         setStakingContract(
-            new StakingContract(_contractAddresses.stakingProxy, providerState.provider, {
+            new StakingContract(_contractAddresses.staking, providerState.provider, {
```

Comment:
Unclear how this ever worked on staging 🤷‍♂

Reply:
🤷‍♂